code/game/machinery/computer/dna_console.dm

```diff
@@ -826,6 +883,46 @@
 						to_chat(usr, "<span class='notice'>You apply [CM] to [HM.name].</span>")
 						stored_chromosomes -= CM
 						CM.apply(HM)
+		if("expand_advinjector")
```

Comment:
Does this thing check if the selected mutation is already in the injector?

Reply:
It doesn't. But there's just no ill effects since it just doesn't do anything.